runtime/interp.c

```diff
@@ -1417,9 +1417,6 @@ do_resume: {
 
 #ifndef THREADED_CODE
     default:
-#ifdef _MSC_VER
-      unreachable();
-#else
```

Comment:
You forgot to remove the matching `#endif` line.

Reply:
You're Lucky Luke! you review my PRs faster than I can test them x)

Reply:
Aren't you supposed to test before `git push`?